qt/qt_common/map_widget.cpp

```diff
@@ -383,8 +383,21 @@ void MapWidget::initializeGL()
     else
     {
       LOG(LINFO, ("Contex is LibGL"));
-      // TODO: Separate apiOpenGL3 from apiOpenGLES3, and use that for the currend shader code.
-      m_apiOpenGLES3 = true;
+      GLint majorVersion = 0, minorVersion = 0;
+      funcs->glGetIntegerv(GL_MAJOR_VERSION, &majorVersion);
+      funcs->glGetIntegerv(GL_MINOR_VERSION, &minorVersion);
+
+      if (majorVersion < 3 || (majorVersion == 3 && minorVersion < 2))
```

Comment:
I also think this might work even on this stage:
```suggestion
      if (context()->format().version() < qMakePair(3, 2))
```

Reply:
I took this one.

Reply:
@Ferenc- Did you check Desktop sill takes `m_apiOpenGLES3 = true` path?

Reply:
Yes it does, as it should.